server_vllm.py

```diff
@@ -318,19 +357,31 @@ async def fake_stream_generator() -> AsyncGenerator[str, None]:
             yield f"data: {response_json}\n\n"
             yield "data: [DONE]\n\n"
 
-        return StreamingResponse(fake_stream_generator(), media_type="text/event-stream")
+        return StreamingResponse(
+            fake_stream_generator(), media_type="text/event-stream"
+        )
 
     return response
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="vLLM OpenAI-Compatible RESTful API server.")
+    parser = argparse.ArgumentParser(
+        description="vLLM OpenAI-Compatible RESTful API server."
+    )
     parser.add_argument("--host", type=str, default="localhost", help="host name")
```

Comment:
I encountered error when using this. Should we change the default back to `0.0.0.0`?

Reply:
AH Ok, I have to change to: 0.0.0.0 too